test/components/canReceiveAndGiveItems.ts

```diff
@@ -0,0 +1,99 @@
+import { IPokemon } from "../../src/components/Battles";
+import { IItemProbabilities } from "../../src/components/mods/RandomHeldItemsMod";
+import { IInventoryListing } from "../../src/components/menus/Items";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+it("Pokemon can be given an item and item is updated in inventory", (): void => {
```

Comment:
These tests typically read like plain English. `it("does something", ...)` and similar. Find a way to phrase the test that way, so it's more understandable in the tests listing.